rules/over-the-shoulder-prs/rule.md

```diff
@@ -0,0 +1,24 @@
+---
+type: rule
+title: Do you do over the shoulder reviews?
+uri: over-the-shoulder-prs
+authors:
+  - title: Brook Jeynes
+    url: https://www.ssw.com.au/people/brook-jeynes
+created: 2023-05-15T21:36:54.519Z
+guid: f20af960-7a60-499c-980b-bd5bb6a0af91
+---
+An "over-the-shoulder" review is one of the best ways to avoid merge debt. When a pull request (PR) is ready to be reviewed, get someone with you either in-person or on call, and go through the PR together. This not only allows you to demo the content of the PR but also talk with the person taking feedback when needed.
+
+A PBI isn't considered complete until the PR has been accepted and merged into the main branch. It is the developer's responsibility to get a PR reviewed before closing the PBI. When a PR is created but left open for a long time, it becomes stale. Stale PRs lead to merge debt because the longer the PR stays open, the more changes occur on the main branch.
```

Comment:
```suggestion
It is the developer's responsibility to get a PR reviewed and action feedback ASAP. When a PR is created but left open for a long time, it becomes stale. Stale PRs lead equate to merge debt because the longer the PR stays open, the more changes occur on the main branch and the harder it is to merge back in.
```